test/extended/operators/resources.go

```diff
@@ -13,8 +13,33 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/kube-openapi/pkg/util/sets"
 	e2e "k8s.io/kubernetes/test/e2e/framework"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
 )
 
+func skipIfWPEnabled(oc *exutil.CLI) {
+	g.By("checking namespaces for WP annotation")
+
+	namespaces, err := oc.KubeFramework().ClientSet.CoreV1().Namespaces().List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		e2e.Failf("unable to list namespaces: %v", err)
+	}
+
+	annotationForAllowingManagementCores := "workload.openshift.io/allowed"
+	annotationValueForAllowingManagementCores := "management"
+	wpEnabled := false
+
+	for _, ns := range namespaces.Items {
+		if val := ns.Annotations[annotationForAllowingManagementCores]; strings.Contains(val, annotationValueForAllowingManagementCores) {
+			wpEnabled = true
+			break
+		}
+	}
+
+	if wpEnabled {
+		e2eskipper.Skipf("test is not relevant on clusters with workload partitioning enabled")
```

Comment:
I think it's still relevant, it's just that the way it's currently written it doesn't work. Maybe the message should be changed to indicate that the test needs to be improved to be made aware of WP

Reply:
I put up another change when fixes up the test to support WP.